### PR TITLE
new distributions for heating/cooling tsps with offsets

### PR DESCRIPTION
**Pull request by aspeake1**
Addresses #[issue number here].

## Pull Request Description

New/updated tsvs for distributions of heating/cooling setpoints and setpoint offsets. Queried from RECS at the AIA climate zone level. Options_lookup is updated with new options.

`Heating/Cooling Setpoint.tsv`: Updated distribution of temperature setpoints 
`Heating/Cooling Setpoint Offset Magnitude.tsv`: Magnitude of offset from the setpoint
`Heating/Cooling Setpoint Offset Period.tsv`: Daily schedule for the offset (night setback, night &day setback, etc.)
`Has Heating/Cooling Setpoing Offset.tsv`: Used as dependency for the offset magnitude and period tsvs

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] The `update_measures` rake task has been run
- [x] The `test:regenerate_osms` rake task has been run
- [x] Unit tests and integrity checks all pass locally
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected circleci regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_multifamily_beta/pat.json [hunk 1]
```diff
@@ -494,7 +494,11 @@
       "seed": "EmptySeedModel.osm",
       "workflow_index": 1,
       "options": [],
+<<<<<<< HEAD
       "instanceId": 0.5261009231493268,
+=======
+      "instanceId": 0.4202025159376823,
+>>>>>>> master
```
Comment (shorowit) [inline project_multifamily_beta/pat.json]: It looks like you committed a bunch of conflicts 👀

Comment (aspeake1) [reply project_multifamily_beta/pat.json]: Thanks, these have been corrected.

**Comment by aspeake1** [comment]
@joseph-robertson 
> 
> 
> I merged master into this branch. And then updated outputs in all pat projects.
> 
> Have you tested this on eagle/aws for how much of an impact it makes on the baseline? Have you checked that the regression tests haven't changed?

Regression tests look good, as does testing on Eagle (500 datapoints)

**Comment by ejhw** [comment]
Whoever merges, please add this feature to the changelog. cc @joseph-robertson

**Comment by joseph-robertson** [comment]
We have a changelog? I thought you guys shot that down... @ejhw

**Pull request merged**